src/Transport.php

```diff
@@ -525,6 +526,9 @@ private function getClientLibraryInfo(): array
         if (false !== strpos($clientClass, 'Symfony\Component\HttpClient')) {
             return ['sy', InstalledVersions::getPrettyVersion('symfony/http-client')];
         }
+        if (false !== strpos($clientClass, 'Elastic\Transport\Client\Curl')) {
+            return ['ec', Transport::VERSION];
+        }
```

Comment:
Would it be good to add "ec" to the list of supported libraries above in the method's comment?